pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/ReactiveMessageReader.java

```diff
@@ -20,10 +20,25 @@
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
+/**
+ * Reactive message reader interface.
+ *
+ * @param <T> the message payload type.
+ * @author Lari Hotari
+ * @author Christophe Bornet
+ */
 public interface ReactiveMessageReader<T> {
 
+	/**
+	 * Read one message.
+	 * @return the message read
+	 */
 	Mono<Message<T>> readOne();
 
+	/**
+	 * Read messages continuously.
+	 * @return the messages read
+	 */
 	Flux<Message<T>> readMany();
```

Comment:
It would be useful to refer to ReactiveMessageReaderBuilder#endOfStreamAction in the documentation for these methods. When endOfStreamAction == EndOfStreamAction#COMPLETE, readOne will return an empty Mono if the reader reaches the end of the stream of messages.   
I'll add a separate issue to document endOfStreamAction.

Reply:
#40